addons/survey/controllers/main.py

```diff
@@ -373,6 +390,40 @@ def survey_get_question_image(self, survey_token, answer_token, question_id, sug
 
         return Binary._content_image_get_response(status, headers, image_base64)
 
+    def _set_survey_conditional_values(self, survey, answer):
+        """ For survey containing conditional questions, we need a conditional_map that contains
```

Comment:
This method is a bit useless as used only once and contains nin grand chose, but this is mainly to have the complete explanation about conditional management before giving everything to the template.